metals/src/main/scala/scala/meta/internal/metals/Compilations.scala

```diff
@@ -106,13 +109,16 @@ final class Compilations(
       .andThen {
         case result =>
           updateCompiledTargetState(result)
-
-          // See https://github.com/scalacenter/bloop/issues/1067
-          classes.rebuildIndex(targets).foreach { _ =>
-            if (targets.exists(isCurrentlyFocused)) {
-              languageClient.refreshModel()
-            }
+      }
```

Comment:
@alekseiAlefirov I think we need to change here too, otherwise we can compile but not fetch classes and try to run.